Game/Materials.md

```diff
@@ -0,0 +1,168 @@
+## Material
+All things are made up of materials. This usually isn't a major concern to the average adventurer, but a savvy tradesman will want to carefully select materials of varying qualities for their projects. Characters will need to source materials for anything they are [Crafting](Crafting). Materials should be simple and interchangeable withing a quality.
+
+### Material Quality
+All materials generally come in three quality levels:
+
+#### Basic
+simple materials, not very useful for crafting anything but the most basic of things.
+#### Artisan
+Well treated, high-quality materials used for proper tools and equipment.
+#### Exotic
+Impressive materials, sourced from unique locations and situations with powerful properties.
+
+### Material Types
+There are generally 8 types of materials that will be used for creating:
+
+#### Bits
+After killing a creature you can, unless it’s been annihilated, usually [harvest](Equipment#Looting) some of what are called bits. Bits are a generalization of organs, glands, bones, and other valuables. Your character would intuitively know what parts of a creature are valuable, even if you don’t yet. 
```

Comment:
Readability change:  "[...] usually harvest some of what are called bits." -> "[...] usually harvest some bits" || "[...] usually harvest bits"